[2025-09-01T04:20:14.324+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: etl_pipeline.archive manual__2025-09-01T04:20:05.597940+00:00 [queued]>
[2025-09-01T04:20:14.331+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: etl_pipeline.archive manual__2025-09-01T04:20:05.597940+00:00 [queued]>
[2025-09-01T04:20:14.332+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 1
[2025-09-01T04:20:14.349+0000] {taskinstance.py:2191} INFO - Executing <Task(PythonOperator): archive> on 2025-09-01 04:20:05.597940+00:00
[2025-09-01T04:20:14.356+0000] {standard_task_runner.py:60} INFO - Started process 803 to run task
[2025-09-01T04:20:14.357+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'etl_pipeline', 'archive', 'manual__2025-09-01T04:20:05.597940+00:00', '--job-id', '9', '--raw', '--subdir', 'DAGS_FOLDER/etl_pipeline.py', '--cfg-path', '/tmp/tmp866b1kzc']
[2025-09-01T04:20:14.360+0000] {standard_task_runner.py:88} INFO - Job 9: Subtask archive
[2025-09-01T04:20:14.378+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.8/site-packages/***/settings.py:194 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2025-09-01T04:20:14.416+0000] {task_command.py:423} INFO - Running <TaskInstance: etl_pipeline.archive manual__2025-09-01T04:20:05.597940+00:00 [running]> on host c7e4bfbd9438
[2025-09-01T04:20:14.498+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='etl_pipeline' AIRFLOW_CTX_TASK_ID='archive' AIRFLOW_CTX_EXECUTION_DATE='2025-09-01T04:20:05.597940+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-09-01T04:20:05.597940+00:00'
[2025-09-01T04:20:14.501+0000] {logging_mixin.py:188} INFO - Running task: ARCHIVE
[2025-09-01T04:20:14.502+0000] {logging_mixin.py:188} INFO - Simulating the archival of the original raw data...
[2025-09-01T04:20:14.503+0000] {python.py:201} INFO - Done. Returned value was: None
[2025-09-01T04:20:14.521+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=etl_pipeline, task_id=archive, execution_date=20250901T042005, start_date=20250901T042014, end_date=20250901T042014
[2025-09-01T04:20:14.589+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2025-09-01T04:20:14.613+0000] {taskinstance.py:3280} INFO - 0 downstream tasks scheduled from follow-on schedule check
